### DISCUSSION TITLE
Cannot set the cookies

### DISCUSSION THREAD

**Discussion by jvictorfranca** [directus/directus#11276] [Archive]
Hi once again!

I am currently trying to get microsoft SSO within Directus. I could make the SSO work for directus, but i cannot seem to get the cookie set for retrieving the access token. Im trying to use directus on localhost:8055 and redirect to localhost:3000 to get the access token. 
My env file is as follows:
AUTH_PROVIDERS="microsoft"
AUTH_DISABLE_DEFAULT ="false"
AUTH_MICROSOFT_DRIVER="openid"
AUTH_MICROSOFT_CLIENT_ID="myCliendId"
AUTH_MICROSOFT_CLIENT_SECRET="mySecret"
AUTH_MICROSOFT_SCOPE="openid profile email"
AUTH_MICROSOFT_IDENTIFIER_KEY="email"
AUTH_MICROSOFT_ICON="microsoft"
AUTH_MICROSOFT_ISSUER_URL="https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration"

#############################################################################################################

REFRESH_TOKEN_COOKIE_DOMAIN="http://localhost:3000" 
REFRESH_TOKEN_COOKIE_SECURE="true"
REFRESH_TOKEN_COOKIE_SAME_SITE="None"

I tried using firefox, and my headers appears to have the set cookie comming, but the cookies in the inspec application are not there at localhost.

![image](https://user-images.githubusercontent.com/83782983/151071595-6350925c-e920-44cf-8253-aff2bd9bd4e1.png)

![image](https://user-images.githubusercontent.com/83782983/151071734-f6d709a7-83a2-4de1-bfff-6aab510f6dc5.png)

Thanks!

**Comment by rijkvanzanten** [comment]
@jvictorfranca I think `REFRESH_TOKEN_COOKIE_SECURE=true` isn't compatible with your domains (`http://localhost:3000`) as it's not HTTPS and thus not "secure" 🤔

**Reply by jvictorfranca** [reply]
I see, thanks. Now im having 2 questions openned, sorry for that... I will mark this one as answered. Thaks again
